src/decorators/swagger/api-common-error-code.swagger.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpStatus, applyDecorators } from '@nestjs/common';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { COMMON_ERROR_CODE } from '@src/constants/error/common/common-error-code.constant';
+import { HttpException } from '@src/http-exceptions/exceptions/http.exception';
+
+export function ApiCommonErrorCode(): ClassDecorator {
```

Comment:
```suggestion
type Statuses = (401 | 403 | 500)[];

const decoratorMap: Record<
  Statuses[number],
  (ClassDecorator | MethodDecorator)[]
> = {
  401: [
    ApiBearerAuth(),
    HttpException.swaggerBuilder(HttpStatus.UNAUTHORIZED, [
      COMMON_ERROR_CODE.INVALID_TOKEN,
    ]),
  ],
  403: [
    ApiBearerAuth(),
    ,
    HttpException.swaggerBuilder(HttpStatus.FORBIDDEN, [
      COMMON_ERROR_CODE.PERMISSION_DENIED,
    ]),
  ],
  500: [
    HttpException.swaggerBuilder(HttpStatus.INTERNAL_SERVER_ERROR, [
      COMMON_ERROR_CODE.SERVER_ERROR,
    ]),
  ],
};

export function ApiCommonResponse(statuses: Statuses = [401, 403, 500]) {
  return applyDecorators(
    ...statuses.flatMap((arg) => {
      return decoratorMap[arg];
    }),
  );
}
```

swagger를 켜봤는데 모든 api에 bearer auth 가 붙어있는게 생각보다 좋아보이지 않더라고요 차라리 예시코드처럼 선택해서 받을 수 있게끔 하는게 나을거같습니다. 예시코드를 꼭 따르지 않아도 됩니다.
그리고 해당 함수는 commonResponse 로 명명하고 common한 response에 대한 제어를 하고
common한 request 같은 애들은 다른 함수로 관리되는 형태로 되는것도 괜찮아보이고요